JBBCode/CodeDefinition.php

```diff
@@ -11,28 +11,28 @@
  */
 class CodeDefinition
 {
-    /* NOTE: THIS PROPERTY SHOULD ALWAYS BE LOWERCASE; USE setTagName() TO ENSURE THIS */
+    /* @var string NOTE: THIS PROPERTY SHOULD ALWAYS BE LOWERCASE; USE setTagName() TO ENSURE THIS */
```

Comment:
Maybe IDEs will recognize it, but phpdoc needs `/**`, see http://phpdoc.org/docs/latest/references/phpdoc/tags/var.html#examples